Documentation/howtogetstarted.md

```diff
@@ -1,6 +1,25 @@
-HOWTO get started with the Reference Security Incident Taxonomy
+# HOWTO get started with the Reference Security Incident Taxonomy
 
-1. Read the [documentation](https://github.com/enisaeu/Reference-Security-Incident-Taxonomy-Task-Force/tree/master/Documentation)
+Using the Reference Security Incident Taxonomy in your environment doesn't require a lot of effort.
+
+1. Read the [documentation](https://github.com/enisaeu/Reference-Security-Incident-Taxonomy-Task-Force/tree/master/Documentation), especially take care of the [use cases](https://github.com/enisaeu/Reference-Security-Incident-Taxonomy-Task-Force/blob/master/Documentation/Use%20Cases.md) as these can give you additional ideas where to use the RSIT.
 2. Check the latest [version](https://github.com/enisaeu/Reference-Security-Incident-Taxonomy-Task-Force/tree/master/working_copy) of the taxonomy, both in human readable format and in machine readable
 3. Update the [usage page](https://github.com/enisaeu/Reference-Security-Incident-Taxonomy-Task-Force/blob/master/Documentation/Dependencies%20and%20tool%20mapping.md) if you use it
-4. Push changes if you see fit
+4. Push changes if you see fit or open an issue if you have questions or comments
+
+## Multiple values
```

Comment:
Could someone from ENISA help my memory, is this explanation for multiple values the one we agreed upon as a group?
Or is it an individual proposal?

Reply:
Hi @aaronkaplan ... my PR was based on the mail from Rossella


> All agree for the multiple values that 1st one is intent (time dependent) and the rest are the transport mechanisms. > All agree to update the doc accordingly before next meeting. Please edit this https://github.com/enisaeu/Reference-Security-Incident-Taxonomy-Task-Force/blob/master/Documentation/howtogetstarted.md

Happy to update the proposal though ...